package.json

```diff
@@ -1,5 +1,21 @@
 {
-  "name": "ux-onboarding-project",
+  "name": "nathan-phillips-ux-iop",
   "version": "1.0.0",
-
+  "main": "index.html",
+  "dependencies": {
+    "server": "^1.0.18"
```

Comment:
I don't think this is quite right. It probably worked locally because you had already installed it correctly locally

I would try deleting this line and then running

```
npm i --save-dev http-server
```

`--save-dev` just adds it to your devDependencies list